### ISSUE TITLE
build_pythia6.sh.txt not accesible offsite

### ISSUE THREAD

**Issue by Hugo-Leung** [E1039-Collaboration/e1039-share#8]
I am trying to build e1039-share offsite. And noticed http://home.fnal.gov/~rhatcher/build_pythia6.sh.txt is behind Fermilab SSO.
Thus 30_pythia6.sh will fail offsite. Do we want to include this file in e1039-share?

**Comment by liuk** [comment]
Hi Hugo, PYTHIA6 is no longer needed for our software. See this PR https://github.com/E1039-Collaboration/e1039-core/pull/108. We should just remove the pythia6 build scripts from this repo.

**Comment by knakano0524** [comment]
@Hugo-Leung ; As Kun wrote, you can just remove `30_pythia6.sh` and run `install-all.sh` for now.  I will update the repository later.

**Comment by Hugo-Leung** [comment]
I will do that, thank you

**Comment by Hugo-Leung** [comment]
how about 32_pythia8.sh? http://home.thep.lu.se/~torbjorn/pythia8/pythia8240.tgz is no longer accessible

**Comment by liuk** [comment]
OK that is new. Probably need to be replaced with https://pythia.org/download/pythia82/pythia8240.tgz.

**Comment by knakano0524** [comment]
Hmm, please modify the script to use `https://pythia.org/download/pythia82/pythia8240.tgz`.

URLs in the scripts sometimes become invalid as the structure of source website has changed.  We are updating the scripts time-by-time.

**Comment by Hugo-Leung** [comment]
So I found some weird behavior with 90_rave.sh and 91_genfit.sh. They both fail at `source $DIR_INST/this-share.sh`.
If I comment out this line and source this-share.sh before running the script, it works fine. 

I am using e1039-sl7 singularity image if that matters

**Comment by knakano0524** [comment]
@Hugo-Leung I encountered no error on gpvm01 using the e1039-sl7 singularity image.  Can you post the (error) messages that 90_rave.sh and 91_genfit.sh outputted?

I have made the modifications about pythia6 and pythia8.

**Comment by Hugo-Leung** [comment]
The error message I got is
```
Script: /home/chleung/SpinQuest/share_v2/script/sub/90_rave.sh
----------------
Abort as '/home/chleung/SpinQuest/share_v2/script/sub/90_rave.sh' returned '5'.
Check details in '/home/chleung/SpinQuest/share_v2/log/log_90_rave.txt'.
```

The log file is empty and I trace it back to `source $DIR_INST/this-share.sh`

**Comment by knakano0524** [comment]
`this-share.sh` could return 5 when `source $FN_THISROOT` or `source $FN_GEANT4` returns 5.  But I don't see how it happens and I cannot reproduce the error.  I'd like to close this issue, as you already have a workaround.  I will reconsider it when running `install-all.sh` next time on gpvm or another machine.

**Comment by Hugo-Leung** [comment]
Sure, we can reopen this later if this happens on gpvm or another machine

**Comment by Hugo-Leung** [comment]
Solved in commit da5a058f1360450b5625889cf5507c1fb4e52ea0
